src/routes/user.js

```diff
@@ -2,21 +2,22 @@ import express from 'express';
 import getUsers, {
   getUser, createUser, updateUser, deleteUser,
 } from './admin/user';
+import userAdminAccessMiddleware from '../middleware/userAdminAccessMiddleware';
 
-const router = new express.Router();
+const router = express.Router();
 
 /**
  * API for frontend to manage user updates.
  */
 
-router.get('/:userId', getUser);
+router.get('/:userId', userAdminAccessMiddleware, getUser);
```

Comment:
Can we use `router.use(userAdminAccessMiddleware)` above the route definitions instead of applying the middleware on each route? One less thing to think about when adding additional routes!